engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -1345,6 +1345,21 @@ class demon_hunter_action_t : public Base
     ab::apply_affecting_aura( p->talent.havoc.any_means_necessary );
     ab::apply_affecting_aura( p->talent.havoc.dancing_with_fate );
 
+    ab::apply_affecting_aura( p->talent.vengeance.perfectly_balanced_glaive );
+    ab::apply_affecting_aura( p->talent.vengeance.meteoric_strikes );
+    ab::apply_affecting_aura( p->talent.vengeance.shear_fury );
+    ab::apply_affecting_aura( p->talent.vengeance.roaring_fire );
+    ab::apply_affecting_aura( p->talent.vengeance.agonizing_flames );
+    ab::apply_affecting_aura( p->talent.vengeance.extended_spikes );
+    ab::apply_affecting_aura( p->talent.vengeance.burning_blood );
+    ab::apply_affecting_aura( p->talent.vengeance.ruinous_bulwark );
+    ab::apply_affecting_aura( p->talent.vengeance.volatile_flameblood );
+    ab::apply_affecting_aura( p->talent.vengeance.chains_of_anger );
+    ab::apply_affecting_aura( p->talent.vengeance.stoke_the_flames );
+    // Only handles the maximum stacks for Frailty
+    ab::apply_affecting_aura( p->talent.vengeance.soulcrush );
```

Comment:
Max stacks have to be applied to the buff. debuffs.frailty->apply_affecting_aura

Reply:
I'll handle that in a separate PR then and address more frailty things at the same time.